pom.xml

```diff
@@ -22,6 +27,20 @@
     <url>http://www.hotels.com</url>
   </organization>
 
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+    </license>
+  </licenses>
+
+  <scm>
+     <connection>scm:git:git://github.com/netty/netty.git</connection>
```

Comment:
Is this reference to a netty git repository intentional?

Reply:
Totally not.